spec/support/shared_examples/name_with_language.rb

```diff
@@ -0,0 +1,14 @@
+shared_examples 'name_with_language' do
+
+  describe "#current_language" do
+    it "returns name_zh_tw for chinese" do
+      I18n.locale = 'zh-tw'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.